### ISSUE TITLE
Pretrained .pth checkpoint

### ISSUE THREAD

**Issue by MaksymHalychShi2022** [instant-high/wav2lip-onnx-HQ#9]
Hello, where can I find pretrained 256 checkpoint of for fine-tuning? 
Thanks

**Comment by instant-high** [comment]
Try to download the original torch checkpoints here:
https://github.com/Rudrabha/Wav2Lip

**Comment by MaksymHalychShi2022** [comment]
Yeah, I tried that, but it has poor quality due to the 96x96 image size. I saw that you were using the model that operates on 256x256 images in this repository: https://github.com/instant-high/wav2lip-onnx-256.

I would be very grateful if you could share the download link for this model. I tried to find it in the Linly-Talker repo you referenced, but I couldn’t locate the download link.

**Comment by instant-high** [comment]
https://modelscope.cn/models/Kedreamix/Linly-Talker/resolve/master/checkpoints/wav2lipv2.pth

**Comment by MaksymHalychShi2022** [comment]
Thank you so much

**Comment by instant-high** [comment]
Maybe you will share your fine-tuned checkpoint if you succeed?

**Comment by MaksymHalychShi2022** [comment]
I’ll post an update if there’s anything to share.

**Comment by henryruhs** [comment]
https://huggingface.co/Pypa/wav2lip384/blob/main/wav2lip384.pth
